test/e2e/storage/testsuites/base.go

```diff
@@ -87,6 +87,10 @@ var BaseSuites = []func() TestSuite{
 	InitStressTestSuite,
 }
 
+// IntreeSuites is a list of storage test suites that work only for in-tree drivers
+var IntreeSuites = append(BaseSuites,
+	InitFsGroupChangePolicyTestSuite)
```

Comment:
We already have a fsgroup driver capability defined. I think we could use that to test fsgroup for CSI drivers right?

If a driver opted into the test capability, we can assume they have their CSIDriver object set correctly.

Reply:
Done. Tested with CSI GCE PD driver. The expectation now is for the driver to setup the Storage class Fstype and CSI Driver object fsGroupPolicy correctly.

Reply:
pull-kubernetes-e2e-gce-csi-serial also runs the fsgroup policy tests on gcepd csi driver